src/screens/PinPad/PinPad.tsx

```diff
@@ -0,0 +1,113 @@
+import { useRef, useState, useEffect, useCallback } from 'react';
+import { SafeAreaView, StatusBar, StyleSheet, View } from "react-native";
+import ReactNativePinView from "react-native-pin-view";
+import BottomDrawer, {
+  BottomDrawerMethods,
+} from 'react-native-animated-bottom-drawer';
+import { Text, Icon } from "@components";
+import { faDeleteLeft, faUnlock } from "@fortawesome/free-solid-svg-icons";
+
+export const PinPad = (props) => {
+  const pinView = useRef(null);
```

Comment:
Above the component, you can add the following:
```
type PinViewFunctions = {
  clearAll: () => void;
};
```

And change this line to
```suggestion
  const pinView = useRef<PinViewFunctions>(null);
```